experimental/generation/readme.md

```diff
@@ -16,31 +16,28 @@ evolving set of capabilities including:
 - Cancel
 - Confirmation
 
-**Ready to [Get Started][start]?** If not, keep reading for more information.
+## Get started ##
+**Ready to get started?** Follow instructions for the [bf-generate plug in][start] or [form dialogs in Composer][link to Getting Started with form dialogs] or keep reading for more information.
```

Comment:
I think this would be a good change to make, i.e. we should change generated dialog to form dialogs.

---
In reply to: [533695284](https://github.com/microsoft/BotBuilder-Samples/pull/2964#discussion_r533695284) [](ancestors = 533695284)